src/server/api/workable.js

```diff
@@ -11,13 +11,12 @@ export default class Workable {
         'Content-Type': 'application/json'
       }
     }).then((response) => {
-      const filtered = response.jobs.map((job) => {
+      return response.body.jobs.map((job) => {
```

Comment:
I'm thinking that maybe fetch shouldn't return status and body but only the data. If the promise gets resolved, we know we're getting the right data because fetch checks the status. In which case this line can just be what it was before this change.